src/orders/utils/data.ts

```diff
@@ -208,3 +210,15 @@ export function mergeRepeatedOrderLines(
     return prev;
   }, Array<OrderDetails_order_fulfillments_lines>());
 }
+
+export const getVariantSearchAddress = (order: OrderDetails_order) => {
+  if (!!order.shippingAddress) {
+    return addressToAddressInput(order.shippingAddress);
+  }
+
+  if (!!order.billingAddress) {
+    return addressToAddressInput(order.billingAddress);
+  }
+
+  return { country: order.channel.defaultCountry.code } as AddressInput;
```

Comment:
Perhaps it would make more sense to type the outcome of this function instead? (assuming it's always AdressInput for all returns)
```
getVariantSearchAddress = (order: OrderDetails_order): AddressInput
```